remoteit/desktop
# [DESK-703] feature logs device

### Description
[DESK-703] feature logs device

### Checklist




### Test plan



1. Sign in
2. Go to Device
3. Click Device Details
4. Click Logs
4. You should  list of logs

### Ticket

https://remoteit.atlassian.net/browse/DESK-703


[DESK-703]: https://remoteit.atlassian.net/browse/DESK-703
